src/wait-for-element.js

```diff
@@ -1,71 +1,21 @@
-import {
```

Comment:
It's easier to just look at the post-change version of this file: https://github.com/testing-library/dom-testing-library/blob/pr/wait-improvements/src/wait-for-element.js